### PR TITLE
Add mobile menu

### PR DESCRIPTION
**Pull request by oyelakinG9**
> In this project, I added JavaScript to make the mobile menu responsive.

### CODE REVIEW

File: css/my_portfolio.css [hunk 1]
```diff
@@ -43,29 +43,156 @@ body {
...
+  left: 0%;
+}
+
+.nav-item {
+  list-style: none;
+  margin: 25px;
+  width: 130%;
+}
+
+.nav-link {
```
Comment (carlos-ssh) [inline css/my_portfolio.css:155]: - Kindly provide the requested design by the style guide on Figma
**_This is the requested design:_**


Screen Shot 2021-12-13 at 16 41 25



**_Yours:_**
Screen Shot 2021-12-13 at 16 40 34

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @oyelakinG9, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/my_portfolio.css (hunk 1 above)
Comment (oyelakinG9) [reply css/my_portfolio.css:155]: @carlos-ssh, Thank you for the comment. Just to inform you nothing with the design. I am sure I met all the requirements. So this is not worth to be rejected.🤞

**Comment by oyelakinG9** [comment]
Yeah! The only issue was `border-bottom` I've tried to extend its length to meet figmal requirement. Thnaks👍

**Comment by oyelakinG9** [comment]
Yeah! The only issue was `border-bottom` I've tried to extend

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @oyelakinG9, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by oyelakinG9** [comment]
> # 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽
> Hi @oyelakinG9,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉
> 
> Approved
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

> Thanks

**Comment by oyelakinG9** [comment]
>Thanks

**Pull request merged**